module/userProjects/helper.js

```diff
@@ -2386,10 +2386,11 @@ module.exports = class UserProjectsHelper {
 	 * @param {String} userId - Logged in user id.
 	 * @param {String} userToken - User token.
 	 * @param {Boolean} isATargetedSolution - User targeted or not .
+	 * @param {String} language - language code
 	 * @returns {Object} Project created information.
 	 */
 
-	static importFromLibrary(projectTemplateId, requestedData, userToken, userId, isATargetedSolution = '') {
+	static importFromLibrary(projectTemplateId, requestedData, userToken, userId, isATargetedSolution = '', language) {
```

Comment:
@praveenKDass please check https://project-elevate-apis.shikshalokam.org/project/v1/userProjects/details api as well

Reply:
Added for details @VISHNUDAS-tunerlabs